test/unit/test_amqpclient.js

```diff
@@ -732,15 +257,15 @@ describe('AMQPClient', function() {
     });
 
     it('should only create a single connection, session, multiple links for multiple receives', function() {
```

Comment:
I'm going to break apart this class, so I'll remove/reword these tests at that point, unless you think its important for this particular PR